# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/webappapis/scripting/events/event-handler-all-global-events.html.ini
# Commit: 1c98db62bb1a
# Full Hash: 1c98db62bb1a20d26f812d28d86672158450e71c
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2024-03-28 21:36:34
# Regressor Bug: 1887729
# File Overlap Count: 2
# Description:
#   Bug 1887729 - Implement context lost/restored support for CanvasRenderingContext2D. r=webidl,gfx-reviewers,smaug,lsalzman
#   
#   Remote canvas can run in the GPU process, and if the GPU process
#   crashes, we need to notify the application using canvas. Historically we
#   just failed, and the application may have been able to continue drawing
# ==============================================================================

diff -r 315bfe14fe3f -r 1c98db62bb1a testing/web-platform/meta/html/webappapis/scripting/events/event-handler-all-global-events.html.ini
--- a/testing/web-platform/meta/html/webappapis/scripting/events/event-handler-all-global-events.html.ini	Thu Mar 28 14:21:58 2024 +0000
+++ b/testing/web-platform/meta/html/webappapis/scripting/events/event-handler-all-global-events.html.ini	Thu Mar 28 14:50:20 2024 +0000
@@ -1,28 +1,4 @@
 [event-handler-all-global-events.html]
-  [oncontextlost: must be on the appropriate locations for GlobalEventHandlers]
-    expected: FAIL
-
-  [oncontextlost: the default value must be null]
-    expected: FAIL
-
-  [oncontextlost: the content attribute must be compiled into a function as the corresponding property]
-    expected: FAIL
-
-  [oncontextlost: the content attribute must execute when an event is dispatched]
-    expected: FAIL
-
-  [oncontextrestored: must be on the appropriate locations for GlobalEventHandlers]
-    expected: FAIL
-
-  [oncontextrestored: the default value must be null]
-    expected: FAIL
-
-  [oncontextrestored: the content attribute must be compiled into a function as the corresponding property]
-    expected: FAIL
-
-  [oncontextrestored: the content attribute must execute when an event is dispatched]
-    expected: FAIL
-
   [onbeforematch: must be on the appropriate locations for GlobalEventHandlers]
     expected: FAIL
 